wordpress-mobile/WordPress-iOS
# Implements the editor title in RN for Gutenberg.

## IMPORTANT:

This is a work in progress.  This PR is NOT ready for merging.

## Description:

Implements the editor title in RN for Gutenberg, also removes the native title.

![titleinreactnative](https://user-images.githubusercontent.com/1836005/50998585-adb7c600-1506-11e9-85e8-496785d86fa0.gif)

## Related PRs:

gutenberg: https://github.com/WordPress/gutenberg/pull/13199
gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/450

## Testing: 

### Test 1:

1. Run the app.
2. Make sure Gutenberg is enabled.
3. Open a Gutenberg post and edit the title.
4. Update the post, close the editor.

Make sure the post title has been udpated.

### Test 2:

1. Run the app.
2. Make sure Gutenberg is enabled.
3. Open a Gutenberg post and edit the title.
4. Close the editor without saving.

Make sure the post title hasn't changed.

### Test 3:

1. Run the app.
2. Make sure Gutenberg is enabled.
3. Create a new post.

Make sure the placeholder shows up and the title is empty.

### Test 4:

Make sure multiline titles work fine.

### Test 5:

Make sure pressing ENTER does not insert a newline.  Later on we can change this to move the focus to the content but that's outside the scope of this PR.
